src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -92,8 +108,13 @@
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
-    public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
-    public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
+    public static final String INVALID_FROM_DESC = " " + PREFIX_FROM + "123-1998"; // '&' not allowed in names
+    public static final String INVALID_TO_DESC = " " + PREFIX_TIME + "123-1998"; // '&' not allowed in names
+    public static final String INVALID_WEBSITE_DESC = " " + PREFIX_WEBSITE + "dfadsf"; // '&' not allowed in names
+    public static final String INVALID_LEVEL_DESC = " " + PREFIX_LEVEL + "basii"; // '&' not allowed in names
+    public static final String INVALID_TYPE_DESC = " " + PREFIX_ITEM + "skil"; // '&' not allowed in names
```

Comment:
Can we fix the comment from `// '&' not allowed in names` to the actual reason why it is invalid?

Reply:
Definitely, my bad